data_steward/deid/config/internal_tables/concept_ids_suppression_files/COPE_conceptsuppressions_conceptids_MaytoJuly2020_FInal.csv

```diff
@@ -1,11 +0,0 @@
-Question,Concept ID,,,,,,,
-o�� Section 2 - COVID Symptoms: Date of onset question,715711,,,,,,,
-o�� Section 3 - COVID Testing: Were you tested for COVID-19 in the past month?,1333327,,,,,,,
-o�� Section 3 - COVID Testing: Was the test for novel coronavirus positive?,1333326,,,,,,,
-o�� Section 3 - COVID Testing: Was the test for COVID-19 positive?,1333326,,,,,,,
-o�� Section 3 - COVID Testing: How were you tested?,1333014,,,,,,,
-o�� Section 3 - COVID Testing: How many times were you tested? (If yes for Were you tested for COVID-19 in the past month),1333118,,,,,,,
-o�� Section 3 - COVID Testing: Were you tested for influenza (flu) in the past month?,1332742,,,,,,,
-"o�� Section 4 - COVID Treatment: In the past month, if you were sick with COVID-19 symptoms, how did you receive treatment(s)?",1333324,,,,,,,
-o�� Section 4 - COVID Treatment: What breathing treatment did you receive? Select all that apply.,1333012,,,,,,,
-o�� Section 4 - COVID Treatment: What other breathing treatment did you receive? Please specify.,1333234,,,,,,,
```

Comment:
Do you think it's better to keep this file in case requirements change to bring it back?

Reply:
The code that gets all concepts gets all the concepts from a few different CSV files in the directory. I thought of moving it to a different location. then thought since it is in git we can always revert back this commit.

Reply:
Yeah, I don't think we have a central location. Since it's not used, I guess it's better to remove it.